modules/cli/pages/index.adoc

```diff
@@ -51,12 +49,257 @@ $ mv skupper $HOME/bin
+----
+
+The default settings include:
 
+* console - The {skupper-name} console is provisioned with a single user.
+The password for the `admin` user is stored in the `skupper-console-users` secret. 
+For more information on the console, see link:{using-console-link}[Using the {skupper-name} console].
+
+* site name - The site name defaults to the namespace name, for example, `west`.
 
 // Type: procedure
 [id="creating-using-cli"] 
-== Creating a Skupper site using the CLI
+== Customizing a site using the CLI
+
+If you require a custom configuration, you might need to recreate the site.
+
+.Procedure
+
+
+. Create a site without a console:
++
+----
+$ skupper delete
+$ skupper init --enable-console false
+----
++
+NOTE: If you have linked to sites, you need to recreate those links after reinitializing the site.
+
+. Configure the {skupper-name} console protection.
+There are a number of `skupper` options regarding authentication for the console:
++
+--
+`--console-auth <authentication-mode>`:: Set the authentication mode for the console:
+
+* `openshift` - Use OpenShift authentication, so that users who have permission to log into OpenShift and view the Project (namespace) can view the console.
+* `internal` -  Use {skupper-name} authentication, see the `console-user` and `console-password` options.
+* `unsecured` - No authentication, anyone with the URL can view the console.
+
+
+`--console-user <username>`:: Username for the console user when authentication mode is set to `internal`.
+`--console-password <password>`:: Password for the console user when authentication mode is set to `internal`.
+--
+
+// Type: procedure
+[id="linking-sites"] 
+== Linking sites
+
+A {service-network} consists of {skupper-name} sites.
+This section describes how to link sites to form a {service-network}.
+
+Linking two sites requires a single initial directional connection. However:
+
+* Communication between the two sites is bidirectional, only the initial linking is directional.
+* The choice of direction for linking is typically determined by accessibility. For example, if you are linking an OpenShift Dedicated cluster with a CodeReady Containers cluster, you must link from the CodeReady Containers cluster to the OpenShift Dedicated cluster because that route is accessible. 
+
+.Procedure
+
+. Determine the direction of the link. If both clusters are publicly addressable, then the direction is not significant. If one of the clusters is addressable from the other cluster, perform step 2 below on the addressable cluster.
+
+. Generate a token on the cluster that you want to link to:
++
+--
+----
+$ skupper token create <filename> 
+----
+
+where `<filename>` is the name of a YAML file that is saved on your local filesystem.
+
+This file contains a key and the location of the site that created it.
+
+NOTE: Access to this file provides access to the {service-network}. Protect it appropriately.
+--
+
+. Use a token to join the {service-network}:
```

Comment:
Maybe it is implicit, but if not just finish this sentence saying:
... on the cluster that will initiate the connection.